tiledb/sm/serialization/query.cc

```diff
@@ -2738,23 +2729,12 @@ Status query_est_result_size_serialize(
       case SerializationType::JSON: {
         ::capnp::JsonCodec json;
         kj::String capnp_json = json.encode(est_result_size_builder);
-        const auto json_len = capnp_json.size();
```

Comment:
Did we loose the behavior of adding \0 here?

Reply:
And I was sure I would miss a case. Fixed, thanks!